Source/WebKit/UIProcess/Inspector/mac/WebInspectorUIProxyMac.mm

```diff
@@ -823,6 +823,38 @@ - (IBAction)_popUpButtonAction:(id)sender
     [inspectedView setFrame:inspectedViewFrame];
 }
 
+static NSArray* transferConstraintsAwayFromInspectedView(NSView* oldConstraintView, NSView* inspectedView, NSView* newContainerView)
+{
+    NSMutableArray* transferredConstraints = [[NSMutableArray alloc] init];
```

Comment:
We'll need to fix this memory leak:

```suggestion
    RetainPtr transferredConstraints = adoptNS([[NSMutableArray alloc] init]);
```

(You might need to use `.get()` in a few places below)

Reply:
Thank for your the review @whsieh ! I will brush up on WebKit memory management and address this as well as the other reviews you left ASAP.